### PR TITLE
Add Honeycomb to the website

### PR DESCRIPTION
**Pull request by akvanhar**
Adds Readme and images with a simple example `main()` function.

### CODE REVIEW

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+{{</highlight>}}
+
+## Viewing your traces
+
+Please visit [honeycomb.io](https://ui.honeycomb.io/) to view your traces
+
+## Project link
+
+You can find out more about the Honeycomb at [https://www.honeycomb.io/](https://www.honeycomb.io/)
```
Comment (emfree) [inline content/guides/exporters/supported-exporters/Go/Honeycomb.md]: nit: s/the Honeycomb/Honeycomb/

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+- [Project link](#project-link)
+
+## Introduction
+
+Honeycomb is a saas tool that provides real-time system debugging. Visualize individual traces to deeply understand request execution. Break down, filter, and aggregate trace data to uncover patterns, find outliers, and understand historical trends.
```
Comment (emfree) [inline content/guides/exporters/supported-exporters/Go/Honeycomb.md]: nit: s/saas/SaaS/, or "Honeycomb is a hosted service that provides..."

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+defer exporter.Close()
+
+    trace.RegisterExporter(exporter)
+
+}
+{{</highlight>}}
+
+## Viewing your traces
+
+Please visit [honeycomb.io](https://ui.honeycomb.io/) to view your traces
```
Comment (emfree) [inline content/guides/exporters/supported-exporters/Go/Honeycomb.md]: nit: maybe `[the Honeycomb UI](https://ui.honeycomb.io/)`, could perhaps also include a link to the tracing docs https://docs.honeycomb.io/working-with-data/tracing/explore-trace-data/ here?

**Review by emfree — APPROVED** [review]
Fantastic. Couple small wording suggestions inline, overall this looks great to me

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md (hunk 2 above)
Comment (christineyen) [inline content/guides/exporters/supported-exporters/Go/Honeycomb.md]: this is the blurb i'm suggesting we use elsewhere, feel free to cut and paste as it feels right: "Honeycomb is a hosted service for debugging your software in production. Capture traces or individual events, then speedily slice and dice your data to uncover patterns, find outliers, and understand historical trends."

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md (hunk 3 above)
Comment (christineyen) [reply content/guides/exporters/supported-exporters/Go/Honeycomb.md]: Yeah. I think that we can do better than the pattern the other exporter docs used ("load localhost:9411," really?).

+1 linking to tracing docs (for **bonus points**, those docs and/or this page might consider linking straight into play.honeycomb.io/tracing)

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+- [Project link](#project-link)
+
+## Introduction
+
+Honeycomb is a saas tool that provides real-time system debugging. Visualize individual traces to deeply understand request execution. Break down, filter, and aggregate trace data to uncover patterns, find outliers, and understand historical trends.
+
+Honeycomb aggregates data at read time, so you don’t have to predict ahead of time which metrics matter. You can fluidly go between time-series graphs, traces, and raw rows to get answers, without having to switch context between tools.
```
Comment (christineyen) [inline content/guides/exporters/supported-exporters/Go/Honeycomb.md]: Let's add a line or two here that tries to differentiate us from the other tracing exporters:

Honeycomb Tracing offers an unparalleled ability to run high-level queries in order to identify **which** traces are worth investigating, before digging deep into the details of an individual exemplar trace. Easily go from insights on your application from a bird's eye view, to a classic waterfall view of a specific trace, and back. For a walkthrough of this flow in action, [visit our blog](https://www.honeycomb.io/blog/2018/07/there-and-back-again-a-honeycomb-tracing-story/).

File: content/guides/exporters/supported-exporters/Go/Honeycomb.md (hunk 3 above)
Comment (christineyen) [reply content/guides/exporters/supported-exporters/Go/Honeycomb.md]: Also, I'll backtrack from my suggestion in Slack to add a "Viewing your metrics" section now that I understand the structure of this site a little more.

And - let's beef up this section to talk a bit more about how this is more than just "view your traces." I'm running out of steam wrt suggested text, but I'd love something that told me, the reader:

Will I just be able to see individual traces? How will I know which traces are interesting? How is this different from the other tracing SaaSes? (Can we include a screenshot of the yet-to-be-built trace summaries page?? Or some updated version of [this gif from honeycomb.io/tracing](https://www.honeycomb.io/wp-content/uploads/2018/06/tracing.gif)?)

**Pull request merged**